velox/docs/functions/presto/string.rst

```diff
@@ -254,3 +254,27 @@ Unicode Functions
 .. function:: to_utf8(string) -> varbinary
 
     Encodes ``string`` into a UTF-8 varbinary representation.
+
```

Comment:
The functions in this file are listed in alphabetical order. Please move the function descriptions to their correct positions.